reference/orders.v3.yml

```diff
@@ -164,7 +164,7 @@ paths:
         * `store_v2_transactions`
 
         **Note:**
-        Order refunds are processed consecutively. Processing synchronous refunds on an order are not yet supported.
+        Create a refund quote before performing a refund request to best avoid a `422` error. Check the refund quote's response body for the `refund_methods` array. The `amount` given in the array must match the `amount` used in the refund request body.
```

Comment:
new content looks good 👍 

Can we also keep the previous one if possible ? that was also a source of confusion for some ENT merchants